packages/react-hooks/src/useRegistrars.ts

```diff
@@ -15,7 +15,7 @@ interface RegistrarNull {
   index: number;
 }
 
-interface Registrar {
+export interface Registrar {
```

Comment:
maybe can be moved to react-hooks/src/types?

Reply:
✅